plugins/arOaiPlugin/modules/arOaiPlugin/templates/_identify.xml.php

```diff
@@ -3,7 +3,7 @@
     <baseURL><?php echo QubitOai::getBaseUrl() ?></baseURL>
     <protocolVersion><?php echo $protocolVersion ?></protocolVersion>
     <?php
-    foreach($adminEmail as $email){
+    foreach($adminEmails as $email){
```

Comment:
Minor style nitpick: the "{" should be moved to the following line (I probably put the incorrect code style in there originally).